src/Components/Opp.tsx

```diff
@@ -14,6 +35,7 @@ const Opp = ({ event }: OppProps ) => {
       <p data-cy='event-time'>Time: {event.start_time} - {event.end_time}</p>
       <p data-cy='event-volunteers'>Volunteers Needed: {event.vols_required}</p>
       <p data-cy='event-description'>Description: {event.description}</p>
+      <DeleteButton />
```

Comment:
adopted similar strat as before by creating a conditionally rendered component to handle the rendering of this delete button. Open to more efficient ideas as always.

Reply:
This is a good way to render the button - it's very clear what the code is trying to do  - nice work!